docs/integrations/mirroring.md

```diff
@@ -79,13 +79,14 @@ def get_remote_data_command(client, args):
 ```
 
 ### get-modified-remote-data
-* GetModifiedRemoteDataArgs - this is an object created to maintain all the arguments you receive from the server in order to use this command.
+* **GetModifiedRemoteDataArgs** - this is an object created to maintain all the arguments you receive from the server in order to use this command.
 Arguments explanation:
-  - last_update - Date string representing the local time that the incident was last updated.
-* GetModifiedRemoteDataResponse - this is the object that maintains the format in which you should order the results from this function. You should use return_results on this object to make it work.
+  - *last_update* - Date string representing the local time that the incident was last updated.
```

Comment:
Not the incident because here we are not running in the context of an incident. `lastUpdate` represents the last time we retrieved modified incidents for this integration.